meta/3rd/wow/config.lua

```diff
@@ -0,0 +1,34 @@
+name    = 'World of Warcraft'
+words   = {'wow.api'}
+files   = {'.*%.toc'}
```

Comment:
AFAIK, Warcraft also uses files with this extension.
And this setting only supports `.lua`

Reply:
I'll try to add common and unique api specific to WoW